### PR TITLE
PIM-7601: Update Symfony to 3.4.4

### PR DESCRIPTION
**Pull request by phaseinducer**
<!--- (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -71,7 +71,7 @@
         "symfony/monolog-bundle": "3.1.0",
         "symfony/swiftmailer-bundle": "3.0.3",
         "symfony/security-acl": "3.0.0",
-        "symfony/symfony": "3.4.2",
+        "symfony/symfony": "3.4.4",
```
Comment (jjanvier) [inline composer.json]: to 3.4.14 or to 3.4.4?

Comment (pierallard) [reply composer.json]: 3.4.14 fails a lot in the CI as far as I read. The minimal need is 3.4.4 for the ticket, so we will try with this one if it's OK for the CI.

File: CHANGELOG-2.3.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 - PIM-7628: Fix the initialization of the product datagrid identifier filter.
 - PIM-7594: Fix memory leak in `pim:versioning:purge` command
+- PIM-7635: Fix elasticsearch config override
```
Comment (pierallard) [inline CHANGELOG-2.3.md]: Why ?

File: composer.json (hunk 1 above)
Comment (jmleroux) [reply composer.json]: I edited the title.

File: CHANGELOG-2.3.md (hunk 2 above)
Comment (jmleroux) [reply CHANGELOG-2.3.md]: Cause the CHANGELOG was incorrect

File: composer.json (hunk 1 above)
Comment (jjanvier) [reply composer.json]: > 3.4.14 fails a lot in the CI

:see_no_evil: :hear_no_evil: 
that's not a good sign :(

Comment (anaelChardan) [reply composer.json]: Wow, shoot :(

Comment (skeleton) [reply composer.json]: do you know on what it fails?

**Pull request merged**